CONTRIBUTING.rst

```diff
@@ -83,9 +83,52 @@ Using a custom ``virtualenv``
     $ cd ${HOME}/hack-on-google-cloud-python
     $ ${VENV}/bin/python setup.py install
 
+- To keep your ``virtualenv`` from getting checked-in, you can make a
+  global gitignore file::
+
+  $ echo "${ENV_NAME}" >> ~/gitignore_global
+  $ git config --global core.excludesfile ~/.gitignore_global
```

Comment:
Up to you, I would like to try and make the contributing path as happy as possible.

Any other suggestions on where to put the venv possibly? `virtualenvwrapper` puts it in `~`.